pkg/ultradns/ultradns.go

```diff
@@ -110,3 +110,80 @@ func (apiConn *APIConnection) Post(url string, body io.Reader) (resp *http.Respo
 	}
 	return resp, err
 }
+
+// Put executes a PUT request at the given url using the APIConnection's client and credentials.
+// This function imitates the http.Post API, but does not require a Content-Type as the type is always set to
```

Comment:
```suggestion
// This function imitates the http.Put API, but does not require a Content-Type as the type is always set to
```

Reply:
(I don't know if "put" or "post" is correct here, but it looked like copypasta?)

Reply:
So, the http library only has Get() Post() and Head() I believe - therefore this should be `Post`
https://golang.org/pkg/net/http/